src/custom/utils/operator/error.ts

```diff
@@ -8,29 +13,23 @@ export enum ApiErrorCodes {
   InsufficientFee = 'InsufficientFee',
   UnsupportedToken = 'UnsupportedToken',
-  WrongOwner = 'WrongOwner'
+  WrongOwner = 'WrongOwner',
+  NotFound = 'NotFound',
+  FeeExceedsFrom = 'FeeExceedsFrom'
 }
 
-export interface ApiError {
-  errorType: ApiErrorCodes
-  description: string
-}
-
-const API_ERROR_CODE_DESCRIPTIONS = {
-  [ApiErrorCodes.DuplicateOrder]: 'There was another identical order already submitted. Please try again.',
-  [ApiErrorCodes.InsufficientFee]:
-    "The signed fee is insufficient. It's possible that is higher now due to a change in the gas price, ether price, or the sell token price. Please try again to get an updated fee quote.",
-  [ApiErrorCodes.InvalidSignature]:
-    'The order signature is invalid. Check whether your Wallet app supports off-chain signing.',
-  [ApiErrorCodes.MissingOrderData]: 'The order has missing information',
-  [ApiErrorCodes.InsufficientValidTo]:
-    'The order you are signing is already expired. This can happen if you set a short expiration in the settings and waited too long before signing the transaction. Please try again.',
-  [ApiErrorCodes.InsufficientFunds]: "The account doesn't have enough funds",
-  [ApiErrorCodes.UnsupportedToken]:
-    'One of the tokens you are trading is unsupported. Please read the FAQ for more info.',
-  [ApiErrorCodes.WrongOwner]:
-    "The signature is invalid.\n\nIt's likely that the signing method provided by your wallet doesn't comply with the standards required by CowSwap.\n\nCheck whether your Wallet app supports off-chain signing (EIP-712 or ETHSIGN).",
-  UNHANDLED_ERROR: 'The order was not accepted by the network'
+export enum ApiErrorCodeDetails {
+  DuplicateOrder = 'There was another identical order already submitted. Please try again.',
+  InsufficientFee = "The signed fee is insufficient. It's possible that is higher now due to a change in the gas price, ether price, or the sell token price. Please try again to get an updated fee quote.",
+  InvalidSignature = 'The order signature is invalid. Check whether your Wallet app supports off-chain signing.',
+  MissingOrderData = 'The order has missing information',
+  InsufficientValidTo = 'The order you are signing is already expired. This can happen if you set a short expiration in the settings and waited too long before signing the transaction. Please try again.',
+  InsufficientFunds = "The account doesn't have enough funds",
+  UnsupportedToken = 'One of the tokens you are trading is unsupported. Please read the FAQ for more info.',
+  WrongOwner = "The signature is invalid.\n\nIt's likely that the signing method provided by your wallet doesn't comply with the standards required by CowSwap.\n\nCheck whether your Wallet app supports off-chain signing (EIP-712 or ETHSIGN).",
+  NotFound = 'Token pair selected has insufficient liquidity',
+  FeeExceedsFrom = 'Fee amount for selected pair exceeds "from" amount',
+  UNHANDLED_ERROR = 'The order was not accepted by the network'
```

Comment:
oh my, the conflicts https://github.com/gnosis/cowswap/pull/737/files

:sob:

Reply:
uh oh

Reply:
meh actually should be fine, doesn't look too bad

Reply:
That's ok, I just like to complain :P